### PR TITLE
Consolidate all translation config

### LINKED ISSUE #3745: Clean up translations
Per this line in this PR https://github.com/apache/incubator-superset/pull/3740/files#diff-21213919e9d7699522fc8d0fcabdc63fR157  it looks like the translations have changed locations, or at least in parts.

I think everything used to be in `babel/` but now it looks like some of it moved to `superset/translations` and some files have stayed where the were.

Should we just move everything under `superset/translations/`?

@graceguo-supercat

### PR DESCRIPTION
**Pull request by alanmcruickshank**
Fixes #3745

Move all translation config to superset/translations, as suggested in the above issue. The `babel` folder which has been removed was being used as a default, hence the update to the docs to show that this now needs to be specified explicitly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -375,6 +376,11 @@ Execute this command to convert the en PO file into a json file:
 
     po2json -d superset -f jed1.x superset/translations/en/LC_MESSAGES/messages.po superset/translations/en/LC_MESSAGES/messages.json
 
+If you get errors running `po2json`, you might be running the ubuntu package with the same
+name rather than the nodejs package (they have a different format for the arguments). You
+need to be running the nodejs version, and so if there is a conflict you may need to point
+directly at `/usr/local/bin/po2json` rather than just `po2json`.
```
Comment (xrmx) [inline CONTRIBUTING.md]: The path would be valid only if you installed with `npm install -g`. The package in debian / ubuntu looks the correct one to latest release 0.4.5 as we have in package.json. What was the error?

Comment (alanmcruickshank) [reply CONTRIBUTING.md]: I did install using using `npm install -g`, as that's what it currently recommends in `CONTRIBUTING.md`. Should we update that to not recommend a global install?

I wonder if actaully I should update the docs here to add clarity around what needs to be done to makes sure `po2json` is installed in the right place more broadly. I found the process quite fiddly for someone who doesn't use node very often - global vs local installs and what folder things are available in was at the root of that I think. I'd love to make the process as easy as:

1. Run command X in folder Y to install Z.
2. Run command A in folder B to convert messages.

Comment (xrmx) [reply CONTRIBUTING.md]: It docs use global install then it's fine.

**Comment by mistercrunch** [comment]
Thanks for adjusting this. Eventually we can wrap those in `npm` and `superset` CLI commands. Though these operations are not very common so it's not too important.

**Pull request merged**